sw/device/tests/sim_dv/BUILD

```diff
@@ -36,13 +36,13 @@ opentitan_functest(
     test_in_rom = True,
     deps = [
         "//hw/top_earlgrey/sw/autogen:top_earlgrey",
+        "//sw/device/lib:pinmux",
```

Comment:
Looks like I missed one. This target doesn't exist anymore. Should depend on //sw/device/lib/dif:pinmux and //sw/device/lib/testing:pinmux_testutils now. Sorry about that!

Reply:
Oh right... this file came into existence after I started the pinmux PR.